packages/components/pie-card/src/index.ts

```diff
@@ -56,6 +56,9 @@ export class PieCard extends LitElement implements CardProps {
     @queryAssignedElements({ flatten: true })
     private assignedElements?: HTMLElement[];
 
+    @property({ type: String, reflect: true, attribute: 'data-test-id' })
+    public dataTestId = 'pie-card';
```

Comment:
I don't think we have this convention at the moment to pass the dataTestId as a prop in our components. Could you please share the use case?

Reply:
Ah, I didn't realise the impact this would have, regarding having to document it etc. Will remove for now, but this is something we may need to add in future. I'll discuss with the team at a later date 👍🏼